.github/workflows/ci.yml

```diff
@@ -33,5 +33,26 @@ jobs:
           cache: 'pnpm'
           cache-dependency-path: '**/pnpm-lock.yaml'
       - run: pnpm install --frozen-lockfile
-      - run: npx playwright install chromium
-      - run: npm run e2e
+      - name: Get installed Playwright version
+        id: playwright-version
+        run: |
+          PLAYWRIGHT_VERSION=$(pnpm playwright -V)
+          echo "PLAYWRIGHT_VERSION=${PLAYWRIGHT_VERSION##Version }" >> $GITHUB_OUTPUT
+      - uses: actions/cache@v3
+        id: playwright-cache
+        if: ${{ inputs.playwright-install == 'true' }}
+        with:
+          path: '~/.cache/ms-playwright'
+          key: '${{ runner.os }}-playwright-${{ steps.playwright-version.outputs.PLAYWRIGHT_VERSION }}'
+          restore-keys: |
+            ${{ runner.os }}-playwright-
+      - name: Install Playwright's dependencies
+        if: steps.playwright-cache.outputs.cache-hit != 'true'
+        run: pnpm playwright install --with-deps
```

Comment:
I wonder if there are simpler solutions, or someone provides actions for playwright caching.
But, it's fine for now.